--- commerce_product.module	2017-09-20 11:14:34.000000000 +0300
+++ commerce_product.module	2017-09-21 18:35:47.000000000 +0300
@@ -99,7 +99,7 @@
   $product = $variables['elements']['#commerce_product'];
 
   $variables['product_entity'] = $product;
-  $variables['product_url'] = $product->toUrl();
+  $variables['product_url'] = ''; // $product->toUrl();
   $variables['product'] = [];
   foreach (Element::children($variables['elements']) as $key) {
     $variables['product'][$key] = $variables['elements'][$key];
@@ -122,7 +122,7 @@
   $product = $product_variation->getProduct();
 
   $variables['product_variation_entity'] = $product_variation;
-  $variables['product_url'] = $product->toUrl();
+  $variables['product_url'] = ''; // $product->toUrl();
   $variables['product_variation'] = [];
   foreach (Element::children($variables['elements']) as $key) {
     $variables['product_variation'][$key] = $variables['elements'][$key];
