--- ProductVariation.php++	2017-09-20 11:14:34.000000000 +0300
+++ ProductVariation.php	2017-09-26 16:21:15.000000000 +0300
@@ -67,15 +67,15 @@ class ProductVariation extends ContentEntityBase implements ProductVariationInte
   public function toUrl($rel = 'canonical', array $options = []) {
     // StringFormatter assumes 'revision' is always a valid link template.
     if (in_array($rel, ['canonical', 'revision'])) {
-      $route_name = 'entity.commerce_product.canonical';
+      $route_name = 'entity.node.canonical';
       $route_parameters = [
-        'commerce_product' => $this->getProductId(),
+        'node' => $this->getProductId(),
       ];
       $options = [
         'query' => [
           'v' => $this->id(),
         ],
-        'entity_type' => 'commerce_product',
+        'entity_type' => 'node',
         'entity' => $this->getProduct(),
         // Display links by default based on the current language.
         'language' => $this->language(),
@@ -385,7 +385,7 @@ class ProductVariation extends ContentEntityBase implements ProductVariationInte
     $fields['product_id'] = BaseFieldDefinition::create('entity_reference')
       ->setLabel(t('Product'))
       ->setDescription(t('The parent product.'))
-      ->setSetting('target_type', 'commerce_product')
+      ->setSetting('target_type', 'node')
       ->setReadOnly(TRUE)
       ->setDisplayConfigurable('view', TRUE);
 
